buildSrc/src/main/kotlin/jni-library.gradle.kts

```
@@ -57,6 +61,12 @@ fun Provider<String>.overrideToString() = object {
     override fun toString() = orNull ?: ""
 }
 
+fun getVariantName(targetMachine: TargetMachine): String {
+    return targetMachine.let {
+        "${it.operatingSystemFamily}-${it.architecture}"
+    }
+}
```

```suggestion
val TargetMachine.getVariantName: String get() = "$operatingSystemFamily-$architecture"
```

+ replace usages `getVariantName(targetMachine)` with `targetMachine.getVariantName`